projects/storefrontlib/shared/components/form/date-picker/date-picker.component.html

```diff
@@ -4,6 +4,7 @@
   [formControl]="control"
   [attr.min]="min"
   [attr.max]="max"
+  [value]="control?.value"
```

Comment:
My point was do we need to add an additional `value` attribute? Instead should we set a value of the control to display a default value I suppose.

Reply:
Yes @Matejk00, the `formControl` value is sufficient.